packages/app-project/pages/projects/[owner]/[project]/logger.js

```diff
@@ -13,8 +13,8 @@ function storeMapper(stores) {
 @inject(storeMapper)
 @observer
 class Logger extends Component {
-  static async getInitialProps({ res }) {
-    const isProduction = process.env.NODE_ENV === 'production'
+  static async getInitialProps({ query = {}, res }) {
+    const isProduction = query.env === 'production' || process.env.NODE_ENV === 'production'
```

Comment:
Apologies I may not have fully understood goal; after `yarn build`, then `yarn start`, then checking http://localhost:3000/projects/brooke/i-fancy-cats/logger?env=staging it's the 404 page and not the "This page is for error testing!" I think because `isProduction` is still being set to `true` due to 'yarn start'. Just double-checking if that's intended.

Reply:
Noting with `yarn dev` then http://localhost:3000/projects/vrooje/kitteh-zoo/logger?env=production it returns a 404 page as I think expected.

Reply:
The staging query param with the production node environment should've overridden it. I've corrected this. Please try rebuilding and checking it again.